### ISSUE TITLE
No default interceptor config

### ISSUE THREAD

**Issue by taylancapraz** [moray95/RestingKit#1]
There must be a configuration for resting kit that automatically add any interceptor as LogInterceptor by default. I don't want to add them in everywhere whenever I initialize restingClient.

**Comment by moray95** [comment]
I don't think it is a "must" but we can use some kind of factory pattern this. You're welcome to open up a pull request 😃
